getting-started/multiband-analysis.rst

```diff
@@ -0,0 +1,293 @@
+  Our goal is to guide the reader through a small data processing project to show what it feels like to use the Science Pipelines.
+  We want this tutorial to be kinetic; instead of getting bogged down in explanations and side-notes, we'll link to other documentation.
+  Don't assume the user has any prior experience with the Pipelines; do assume a working knowledge of astronomy and the command line.
+
+#################################################################################
+Getting started tutorial part 6: analyzing measurement catalogs in multiple bands
+#################################################################################
+
+In this part of the :ref:`tutorial series <getting-started-tutorial>` you'll analyze the forced photometry measurement catalogs you created in :doc:`step 5 <photometry>`.
+You'll learn how to work with measurement tables and plot color-magnitude diagrams (CMDs).
+
+Set up
+======
+
+Pick up your shell session where you left off in :doc:`part 5 <photometry>`.
+That means your current working directory must *contain* the :file:`DATA` directory (the Butler repository).
+
+The ``lsst_distrib`` package also needs to be set up in your shell environment.
+See :doc:`/install/setup` for details on doing this.
+
+As in :doc:`part 3 <display>`, you'll be working inside an interactive Python session for this tutorial.
+You can use the default Python shell (:command:`python`), the `IPython shell`_, or even run from a `Jupyter Notebook`_.
+Ensure that this Python session is running from the shell where you ran :command:`setup lsst_distrib`.
+
+Loading forced photometry measurement catalogs with the Butler
+==============================================================
+
+The :command:`forcedPhotCoadd.py` command-line task (:ref:`getting-started-tutorial-forced-coadds`) created ``deepCoadd_forced_src`` datasets for each coadd in the example data repository.
+Being forced photometry catalogs, rows in each ``deepCoadd_forced_src`` table correspond row-for-row across all coadds in different filters for a sky map patch.
+Since you don't have to do additional cross-matching, these ``deepCoadd_forced_src`` datasets are convenient.
+
+To get these datasets, open a Python session (`IPython`_ or `Jupyter Notebook`_) and create a Butler:
+
+.. code-block:: python
+
+   import lsst.daf.persistence as dafPersist
+   butler = dafPersist.Butler(inputs='DATA/rerun/coaddForcedPhot')
+
+This Butler is using the ``coaddForcedPhot`` rerun you created for the :command:`forcePhotCoadd.py` command-line task's outputs.
+
+Next, use the Butler to get the ``deepCoadd_forced_src`` datasets for both filters:
+
+.. code-block:: python
+
+   rSources = butler.get('deepCoadd_forced_src', {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
+   iSources = butler.get('deepCoadd_forced_src', {'filter': 'HSC-I', 'tract': 0, 'patch': '1,1'})
+
+These datasets correspond to the central patch (``1,1``) coadd for both the HSC-R and HSC-I filters.
```

Comment:
Is this really a 3x3 tract?  Was it constructed to be that way?  Saying "central patch" implies that.  If it's not really the central patch, it's probably better to just say "an example patch" or something.